requirements.txt

```diff
@@ -4,4 +4,6 @@ flask-migrate
 flask-jwt-extended
 flask-argon2
 python-frontmatter
-pycrypto
\ No newline at end of file
+pycrypto
+PyMySQL
+psycopg2-binary
```

Comment:
Does it need to be `psycopg2-binary` or just `psycopg2`?

Reply:
psycopg2 will try to compile, in that way it will be time costy and need to install a bunch of c-dev packages, while psycopg2-binary offers you a compiled one, so why not?

Reply:
That works for me, thanks